src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -145,8 +189,10 @@ string GetPlatformCode()
 #endif
 
         ActiveLicenseFindResult result;
+        readonly Func<DateTime> utcDateTimeProvider;
 
         static ILog Logger = LogManager.GetLogger(typeof(LicenseManager));
         static readonly bool debugLoggingEnabled = Logger.IsDebugEnabled;
+        static readonly TimeSpan ExpirationWarningThreshold = TimeSpan.FromDays(3);
     }
-}
+}
```

Comment:
😱 😉

Reply:
?

Reply:
Just playing white space 👮 . The EOL should not be removed. 🙂

Reply:
tell that visual studio ;)

Reply:
Is it doing this automatically?

Reply:
I'm not going to play with these characters for fun ;)